### PR TITLE
Switch to response files for GenFacades

### PR DESCRIPTION
**Pull request by weshaggard**
cc @ericstj @mellinoe @joperezr

### CODE REVIEW

File: src/common/CommandLine.cs [hunk 1]
```diff
@@ -590,11 +590,18 @@ public bool LastQualifierWins
...
         }
         public CommandLineParser(string[] args)
         {
-            _args = new List<string>(args);
+            _args = new List<string>();
+            _seenExeArg = true;
+
+            for (int i = 0; i < args.Length; i++)
+            {
+                ParseWords(args[i]);
```
Comment (ericstj) [inline src/common/CommandLine.cs]: Isn't this double-parsring the args? Suppose you had nested quotes with something like `"\"Hello\" \"World\""`.  Previously you'd get a single _arg entry as `"Hello" "World"`.  Now you'll get two entries: `Hello` and `World`.

**Comment by mellinoe** [comment]
What's the value? I usually prefer to have the arguments right there in the command, I find it easier to debug, easier to modify, etc.

File: src/common/CommandLine.cs (hunk 1 above)
Comment (weshaggard) [inline src/common/CommandLine.cs]: Fair maybe I will just parse the args that start with "@".

**Comment by weshaggard** [comment]
>What's the value? I usually prefer to have the arguments right there in the command, I find it easier to debug, easier to modify, etc.

I'm only fixing this because I've hit the command line limit while working on netstandard and there isn't any other way to handle it besides switching to a response file. Also for me personally I've always found it easier to repro issues with a response file because it is a lot easier to run most of things commands by just passing in that one argument.

**Comment by ericstj** [comment]
LGTM

**Pull request merged**